### PR TITLE
fixed-send-button-being-active-without-input

### PR DESCRIPTION
**Pull request by Raju-github-profile**
#What does this PR do?
FIxes issue #6940
Enhance "Send" button behavior in Suggest Datasource

### CODE REVIEW

File: frontend/src/Editor/DataSourceManager/DataSourceManager.jsx
Comment (Ckbhatia) [inline frontend/src/Editor/DataSourceManager/DataSourceManager.jsx:931]:
```suggestion
              <Button className="mt-2" disabled={!inputValue.length} variant="primary" onClick={handleSend}>
```

Comment (Ckbhatia) [reply frontend/src/Editor/DataSourceManager/DataSourceManager.jsx:931]: No need to write explicitly.

Comment (Raju-github-profile) [reply frontend/src/Editor/DataSourceManager/DataSourceManager.jsx:931]: Yes !! Even that can be done!!

**Review by Ckbhatia — COMMENTED** [review]
Looks all good from the side.

@Navaneeth-pk needs the maintainers' review & approval.

**Comment by Raju-github-profile** [comment]
@Navaneeth-pk @uhlenbrock  @frafra  needs the maintainers' review & approval.

**Comment by frafra** [comment]
I am not a maintainer. Please do not mention random people. I wish you the best for your PR.

**Review by kavinvenkatachalam — APPROVED** [review]

**Pull request merged**
